sunpy/timeseries/timeseriesbase.py

```diff
@@ -124,9 +124,9 @@ def source(self):
         return self._source
 
     @property
-    def columns(self):
+    def colnames(self):
         """A list of all the names of the columns in the data."""
-        return list(self.data.columns.values)
+        return list(self.data.colnames.values)
```

Comment:
I'm not sure this needs to be changed here. You have to check what self.data is. 

The same goes for any other places too.

Reply:
Okay I will check this...let me know the type to be mentioned in changelog too

Reply:
I forgot to mention you could try running just the `timeseries` tests by passing the path to them to `pytest`. That way it should work about the other issue you had.

Reply:
Okay so I tried running `pytest `and I get this error
`pytest: error: unrecognized arguments: --doctest-rst`  Is this because the docs are not updated yet?

Reply:
You are missing packages for our tests. Install pytest-astropy and that should work.